ee/modules/030-cloud-provider-openstack/templates/cloud-controller-manager/secret.yaml

```diff
@@ -43,6 +43,7 @@ floating-network-id = {{ .Values.cloudProviderOpenstack.internal.loadBalancer.fl
 enable-ingress-hostname = true
   {{- end }}
 [BlockStorage]
+ignore-volume-microversion = {{ .Values.cloudProviderOpenstack.internal.ignoreVolumeMicroversion }}
```

Comment:
You should add kubernetes version check, because it option exists only in cinder-csi-driver for k8s 1.23 and 1.24.